ictv/plugins/editor/api.py

```diff
@@ -73,6 +74,11 @@ def DELETE_AUTH(self, *args, **kwargs):
         raise web.nomethod()
 
 
+class APITemplates(EditorAPIPage):
+    def GET_AUTH(self, channel):
+        return Templates._templates
```

Comment:
Using the "public" interface of `Templates` instead of how it is implemented would be cleaner IMO, e.g.:
 `return { k: Templates[k] for k in Templates}`